src/Microsoft.DotNet.Interactive/Extensions/KernelUriExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+
+namespace Microsoft.DotNet.Interactive.Extensions
```

Comment:
This namespace is intended to be about kernel extensions, not extension methods.

Reply:
Moved to `Microsoft.DotNet.Interactive.Utility` alongside other extension methods.